docs/appendices/release-notes/unreleased.rst

```diff
@@ -97,6 +97,9 @@ Changes
 - Improved the performance of the :ref`hyperloglog_distinct
   <aggregation-hyperloglog-distinct>` aggregation function.
 
+- Improved count aggregations on object types containing not-null constrained
+  inner types by utilizing the inner types' doc-values when available.
+
```

Comment:
```suggestion
- Added an optimization that improves the performance of `count()` aggregations on object columns if the object column has at least one inner column with a `NOT NULL` constraint.
```

I think this way it is easier to understand for end-user. Mentioning the doc-values here is probably too much implementation detail.

Reply:
- Added an optimization that improves the performance of `count()` aggregations
  on object columns that have at least one inner column with a `NOT NULL`
  constraint.

Reply:
changed thanks